cypress/support/constants/constants.js

```diff
@@ -400,7 +401,15 @@ module.exports = {
   FCS_DEFAULTTESTDATA_PATH: './cypress/fixtures/defaultTestData.json',
   ENV_SETUP_STATUS: 'environmentLaunched',
   APP_LAUNCH_STATUS: 'appLaunched',
+  VISIBILITYSTATE_VALIDATION_REQ: 'Lifecycle visibility state validation Req # ',
+  LIFECYCLE_VISIBILITYSTATE_SKIP_MESSAGE:
+    '`App is not reachable to fetch visibility state. Skipping Visibility state validation.`',
+  VISIBILITYSTATE_FAILURE_FIX_LOG:
+    '`, How to fix: If the visibility state response is correct for the current platform, but the expected value needs to be changed, add the expected value in configModule file ( cypress.config.js) in the format visibilityState: { ${state} : ${result.report}}`',
+  VISIBILITYSTATE_FAILURE_LOG:
+    '`. If the visibility state response is correct for the current platform, but the expected value needs to be changed, add the expected value in configModule file ( cypress.config.js) in the format visibilityState: { ${state} : ${result.report}}`',
   NO_EVENT_TRIGGERED: 'Expecting no event to be triggered from platform',
+  VISIBILITYSTATE: 'visibilityState',
```

Comment:
There are two "VISIBILITYSTATE" constants. Here and L356. Is this on purpose?

Reply:
Removed one "VISIBILITYSTATE" constants from L356